SQLQuery_ZooManagement.sql

```diff
@@ -0,0 +1,164 @@
+CREATE Database ZooManagement;

```

Comment:
I cannot add comments to the jpg file so I will do it here.
The background is not really helping me but I don't think I see the relationships for Species, Visitors and Tickets. Also, Caretakers and ZooMentenance seems to have a column named "Item3". :D